index.js

```diff
@@ -52,8 +52,13 @@ function basepath(config, service, opts) {
 
 function cljsLambdaBuild(serverless, opts) {
   const fns = slsToCljsLambda(serverless.service.functions, opts);
+  const compiler = _.get(serverless.service, 'custom.cljs-compiler');
```

Comment:
Hyphenated keys are going to look out of place in a serverless.yml.  Please use `lumo: true`.